libs/platform/src/lib/approval-flow/approval-flow-user-list/approval-flow-user-list.component.ts

```diff
@@ -77,6 +80,10 @@ export class ApprovalFlowUserListComponent implements AfterViewInit {
         }
     }
 
+    ngOnChanges(): void {
+        this._collectDataProgressive();
```

Comment:
is it reasonable to run this on each invocation of this hook? I believe you need to stick to `users` input, right?
```suggestion
    if (changes.users) {
        this._collectDataProgressive();
    }
```